Makefile.nt

```diff
@@ -18,8 +18,62 @@ include Makefile.shared
 defaultentry:
 	@echo "Please refer to the installation instructions in file README.win32.adoc."
 
+REQUIRE_FLEXDLL:=$(if $(shell which flexlink 2>/dev/null),,$(FLEXDLL_SRC)/flexlink$(EXE))
+BOOT_FLEXLINK_CMD=$(if $(REQUIRE_FLEXDLL),FLEXLINK_CMD="../boot/ocamlrun ../$(FLEXDLL_SRC)/flexlink.exe")
+# Sys.command doesn't accept / in PATHs (because cmd doesn't)
```

Comment:
Since OCAML_FLEXLINK is interpreted by OCaml code, the same code could easily rewrite / to \ itself (knowing that flexlink does not take /-options), no?  This would avoid these external calls to sed in several Makefiles.